queries/zathurarc/injections.scm

```diff
@@ -0,0 +1,9 @@
+(set_directive
+  (option) @_option
+  (string) @injection.content)
+(#eq? @_option "synctex-editor-command")
+(#offset! @injection.content 0 1 0 -1)
+(#set! injection.language "bash")
+)
```

Comment:
remember to run `./scripts/format-queries.lua queries` before committing

Reply:
I run but nothing changed :cry:

Reply:
Oh you wrote the wrong query ...